.github/workflows/reusable-revised-its.yml

```diff
@@ -42,6 +42,21 @@ on:
         required: false
         type: string
         default: com.mysql.jdbc.Driver
+      DRUID_CLOUD_BUCKET:
+        required: false
+        type: string
+      DRUID_CLOUD_PATH:
+        required: false
+        type: string
+      AWS_REGION:
+        required: false
+        type: string
+      AWS_ACCESS_KEY_ID:
+        required: false
+        type: string
+      AWS_SECRET_ACCESS_KEY:
+        required: false
+        type: string
```

Comment:
Not covered here: if we no longer need env vars for S3, please remove the checks for them in 'it.sh`.

Reply:
We still need these vars for S3. It is provided in [revised-its.yml](https://github.com/apache/druid/pull/13997/files/dffa60d450b2858965a9f8196fae789a185c7ef4#diff-70af0d1891a4b15a7fa2c1a0f19c26cfb916b23af84856df56ef7090252dbb38) for `s3-deep-storage-minio`.

`required: false` here refers the value being not mandatory to be provided (only needed by s3 tests), as we use the same `reusable-revised-its.yml` workflow for all ITs